python/cuml/svm/svc.pyx

```diff
@@ -52,6 +52,84 @@ if has_sklearn():
     from cuml.multiclass import MulticlassClassifier
     from sklearn.calibration import CalibratedClassifierCV
 
+
+def apply_class_weight(handle, sample_weight, class_weight, y, verbose, output_type, dtype) -> CumlArray:
```

Comment:
Small nitpick also, but this is a Python function, it'd be nicer to put it after the Cython ones for readability of the file maybe?